src/controller/python/chip/clusters/Attribute.py

```diff
@@ -520,6 +520,22 @@ def GetReportingIntervalsSeconds(self) -> Tuple[int, int]:
 
         return minIntervalSec.value, maxIntervalSec.value
 
+    def GetSubscriptionTimeout(self) -> int:
```

Comment:
Please add unit of measurement. `GetSubscriptionTimeoutMs` and describe it in the documentation as well.

Otherwise typical timeouts in python for things like sleep is "float as seconds"

Reply:
Ok